client/components/social-buttons/google.js

```diff
@@ -34,39 +34,37 @@ export default class GoogleLoginButton extends Component {
 		}
 
 		return new Promise( resolve => {
-			loadScript( 'https://apis.google.com/js/platform.js', () => resolve( window.gapi ) );
+			loadScript( 'https://apis.google.com/js/api.js', () => resolve( window.gapi ) );
```

Comment:
I changed it because I believed that platform.js is an older version of the API, but I might be mistaken.

I believe that `api.js` provides promises as return value from it's functions such as `gapi.client.init` while `platform.js` uses callback style.

I was using this documentation:
https://developers.google.com/api-client-library/javascript/start/start-js

Haven't thought about checking the size, thank you @drewblaisdell for catching that! =)